site/docs/components/overview.mdx

```diff
@@ -6,15 +6,13 @@ tags: ["Overview"]
 needToKnow:
 - Components are reusable code used to rapidly build pages. They are the source of truth for our shared User Interface elements.
 - These components are flexible, and we continuously develop and grow them to match our evolving product needs.
+- You can see an overview on the health status of all of the Kaizen components at [https://cultureamp.design/component_health](/component_health)
```

Comment:
😬 
![Screen Shot 2021-08-13 at 2 29 48 pm](https://user-images.githubusercontent.com/1621182/129305000-ce514e39-5156-4e9b-92e8-0d2b8f7ac855.png)

Reply:
```suggestion
- You can see an overview on the health status of all of the Kaizen components at [/component health](https://cultureamp.design/component_health)
===

Reply:
This is what happens when you speed code with laptop battery running out 😅

Reply:
Oh, that doesn't work either because this is frontmatter 🤔 
![image](https://user-images.githubusercontent.com/1811583/129317163-326845fa-c42b-4823-92c0-f6e49bd8deff.png)

Reply:
Ah, bugger